src/main/scala/edu/berkeley/ce/rockslicing/Block.scala

```diff
@@ -335,8 +394,10 @@ case class Block(center: (Double,Double,Double), faces: Seq[Face]) {
           Delaunay.Vector2(rotatedVertex(0), rotatedVertex(1))
         }.distinct.toList
 
-        // If z-component of normal is negative, order needs to be reversed to maintain clockwise orientation
-        if (face.c < -NumericUtils.EPSILON) {
+        // If normal is -z-axis, order needs to be reversed to maintain clockwise orientation since
+        // rotation matrix is identity matrix in this case - vectors are parallel. Rotation matrix
```

Comment:
The rotation matrix will be the identity when vectors are parallel, whether they point in opposite directions or the same direction. The only time we need to change the order is when the normal is the -z-axis since the ordering will end up being clockwise instead of counterclockwise. We're rotating all points so they lie in the z-plane - this allows us to do a 2-D triangulation instead of 3-D. The two special cases where the triangulation will use the points as they are without rotating them is when the normal of the plane is the +/- z-axis. For the +z-axis, everything is fine, but we need to reverse the order for the -z-axis since the points will be ordered as if the normal was the +z-axis. 
Does that make sense? I can add more comments about this, but not sure how much detail is necessary.